[Bug] Crash: isInsideDirectoriesRemovedWithTheApp

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.ichi2.anki/com.ichi2.anki.DeckPicker}: java.lang.NullPointerException: it must not be null
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3707)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3864)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2253)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7870)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
Caused by: java.lang.NullPointerException: it must not be null
	at com.ichi2.anki.ui.windows.managespace.FileUtilsKt.isInsideDirectoriesRemovedWithTheApp(FileUtils.kt:63)
	at com.ichi2.anki.servicelayer.ScopedStorageService.collectionWillBeMadeInaccessibleAfterUninstall(ScopedStorageService.kt:34)
	at com.ichi2.anki.servicelayer.ScopedStorageService.migrationStatus(ScopedStorageService.kt:30)
	at com.ichi2.anki.DeckPicker.startingStorageMigrationInterruptsStartup(DeckPicker.kt:3)
	at com.ichi2.anki.DeckPicker.handleStartup(DeckPicker.kt:45)
	at com.ichi2.anki.DeckPicker.onCreate(DeckPicker.kt:185)
	at android.app.Activity.performCreate(Activity.java:8057)
	at android.app.Activity.performCreate(Activity.java:8037)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1341)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3688)
	... 12 more
```

--- a/AnkiDroid/src/main/java/com/ichi2/anki/ui/windows/managespace/FileUtils.kt
+++ b/AnkiDroid/src/main/java/com/ichi2/anki/ui/windows/managespace/FileUtils.kt
@@ -134,10 +134,10 @@ private suspend fun Context.getUserDataAndCacheSizeUsingGetPackageSizeInfo(): Lo
 fun File.isInsideDirectoriesRemovedWithTheApp(context: Context): Boolean {
     infix fun File.isInsideOf(parent: File) = this.canonicalFile.startsWith(parent.canonicalFile)
 
-    return context.getExternalFilesDirs(null).any { this isInsideOf it } ||
-        context.externalCacheDirs.any { this isInsideOf it } ||
-        context.externalMediaDirs.any { this isInsideOf it } ||
-        context.obbDirs.any { this isInsideOf it } ||
+    return context.getExternalFilesDirs(null).filterNotNull().any { this isInsideOf it } ||
+        context.externalCacheDirs.filterNotNull().any { this isInsideOf it } ||
+        context.externalMediaDirs.filterNotNull().any { this isInsideOf it } ||
+        context.obbDirs.filterNotNull().any { this isInsideOf it } ||
         context.externalDirs.any { this isInsideOf it }
 }
 
@@ -149,6 +149,7 @@ fun File.isInsideDirectoriesRemovedWithTheApp(context: Context): Boolean {
  */
 private val Context.externalDirs: Set<File> get() =
     (getExternalFilesDirs(null) + externalCacheDirs)
+        .filterNotNull()
         .mapNotNullTo(mutableSetOf()) { externalFilesOrCacheDir ->
             externalFilesOrCacheDir.parentFile?.let { parentDir ->
                 if (parentDir.name == packageName) parentDir else null

